openedx/core/djangoapps/ace_common/template_context.py

```diff
@@ -50,8 +59,7 @@ def get_base_template_context(site):
         ),
         'contact_email': get_config_value_from_site_or_settings(
             'CONTACT_EMAIL', site=site, site_config_name='contact_email'),
-        'contact_mailing_address': get_config_value_from_site_or_settings(
```

Comment:
`get_config_value_from_site_or_settings` is not picking the right website which is weird but for now, we can live with it.